src/components/AsideMenu/AsideMenuItem.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+const AsideMenuItem = props => {
+  const { info } = props;
+
+  const [isOpen, setIsOpen] = useState(false);
+  const onClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const detailList = info.Detail.map(data => (
+    <div key={data.id}>{data.Text}</div>
+  ));
+
+  const detailClick = () => {
+    console.log(info.Detail[0]);
+  };
+  return (
+    <>
+      <div className="borderBottom" onClick={onClick}>
+        {info.Text}
+      </div>
+      <div
+        className={'menu ' + (isOpen ? 'openMenu' : 'closeMenu')}
+        onClick={detailClick}
+      >
+        {detailList}
+      </div>
```

Comment:
```suggestion
      <div className="borderBottom" onClick={onClick}>
        {info.Text}
      </div>
      <div
        className={'menu ' + (isOpen ? 'openMenu' : 'closeMenu')}
        onClick={detailClick}
      >
        {info.Detail.map(data => (
    <div key={data.id}>{data.Text}</div>
  ))}
      </div>
```
이 방식이 더 직관적일 것 같습니다.